### PR TITLE
Porting scaffolding over to playbook

### PR DESCRIPTION
**Pull request by JonathanBruce**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/circleci-sidecat
```
Comment (sanghun89) [inline .circleci/config.yml]: `~/circleci-playbook-client`

File: .yarnrc [hunk 2]
```diff
@@ -0,0 +1 @@
+--install.pure-lockfile true
```
Comment (sanghun89) [inline .yarnrc]: you might need to turn this to false first to commit the lock file

Comment (sanghun89) [reply .yarnrc]: oh nvm

Comment (JonathanBruce) [reply .yarnrc]: you do, I did

**Pull request merged**